### ISSUE TITLE
TF 2.0 How to set a different learning rate for each layer?

### ISSUE THREAD

**Issue by yw155** [tensorflow/tensorflow#33036]
## Description of issue (what needs changing):
Hi, I would like to ask a question about how to set different learning rates for several layers. In my implementation, I would like to set one learning rate for some layers while the other learning rate for some other layers. Is there some example codes? Thanks in advance.

**Comment by zhen8838** [comment]
Maybe you can refer to this article and use tf.keras to implement it. 
This is a Chinese blogger's article : https://spaces.ac.cn/archives/6418

**Comment by gadagashwini** [comment]
@yw155, Please provide the Tensorflow version and also provide the code snippet. 
Did you try the @zhen8838's suggestion. Thanks!

**Comment by yw155** [comment]
Hi @gadagashwini, I would like to implement this function in TensorFlow 2.0. I am still not clear how to implement it. 

The function should be able to implement the layer-wise learning rate as follows:
layer {
  name: "conv1"
  type: "Convolution"
  bottom: "input"
  top: "conv1"
  param {
    lr_mult: 1.0
    decay_mult: 1.0
  }
  param {
    lr_mult: 2.0
    decay_mult: 0.0
  }
  convolution_param {
    num_output: 128
    pad: 1
    kernel_size: 3
    weight_filler {
      type: "gaussian"
      std: 0.01
    }
    bias_filler {
      type: "constant"
    }
  }
}
layer {
  name: "conv1"
  type: "Convolution"
  bottom: "input"
  top: "conv1"
  param {
    lr_mult: 4.0
    decay_mult: 1.0
  }
  param {
    lr_mult: 8.0
    decay_mult: 0.0
  }
  convolution_param {
    num_output: 128
    pad: 1
    kernel_size: 3
    weight_filler {
      type: "gaussian"
      std: 0.01
    }
    bias_filler {
      type: "constant"
    }
  }
}

**Comment by jvishnuvardhan** [comment]
This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!
